mlip_testing/calcs/surfaces/cat_surfaces.py

```diff
@@ -0,0 +1,81 @@
+"""Surface benchmark category node."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import zntrack
+
+from mlip_testing.calcs.OC157.calc_OC157 import OC157Benchmark
+from mlip_testing.calcs.S24.calc_S24 import S24Benchmark
+
+
+class SurfaceCategory(zntrack.Node):
+    """
+    Category node for surface-related benchmarks.
+
+    Aggregates OC157 and S24 benchmarks into a single category.
+    """
+
+    oc157_benchmarks: list[OC157Benchmark] = zntrack.deps()
+    s24_benchmarks: list[S24Benchmark] = zntrack.deps()
+
+    def run(self):
+        """Categories don't run calculations - they just organize the DAG."""
```

Comment:
```suggestion
        """Organise the DAG (categories don't run calculations)."""
```
For docstring consistency ("Summary must start with infinitive verb")